docs/ToolbarMenu.md

```diff
@@ -0,0 +1,79 @@
+# Toolbar Menu
+
+The `ToolbarMenu` component is used to display a dropdown menu with label. The Menu can be populated via the `menuGroups` prop, or can be entirely customized by supplying your own `<Menu>` via the `menu` prop.
+
+## Usage
+
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 600px" alt="Toolbar Menu" src="./images/ToolbarMenuAnatomy.png">
+</div>
+
+```tsx
+import { ToolbarMenu } from '@brightlayer-ui/react-components';
+...
+const useStyles  = makeStyles((theme: Theme) =>
+    createStyles({
+            labelContent: {
+            '& > span': {
+                display: 'flex',
+            }
+        },
+        icon: {
+            marginRight: `${theme.spacing(1)}px`,
+        },
+    })
+);
+
+<ToolbarMenu
+    label={
+        <span className={classes.labelContent}>
+            <GradeA className={classes.icon} />
+            <span>Subtitle</span>
+        </span>
+    }
+    menuGroups={[
+      {
+        items: [{
+            title: "Menu Item 1";
+            onClick: () => {};
+          },
+          {
+            title: "Menu Item 2";
+            onClick: () => {};
+          },
+          {
+            title: "Menu Item 3";
+            onClick: () => {};
+          }
+        ]
+      }
+    ]}
+/>
+```
```

Comment:
The first example in the Readme should be the most basic example possible with the minimum amount of excess imports or functionality:
```suggestion
```tsx
import { ToolbarMenu } from '@brightlayer-ui/react-components';
...
<ToolbarMenu
    label={'label'}
    menuGroups={[
      {
        items: [{
            title: "Menu Item 1";
            onClick: () => {};
          },
          {
            title: "Menu Item 2";
            onClick: () => {};
          },
        ]
      }
    ]}
/>
```
```